src/liquid/components/ld-tabs/ld-tablist/ld-tablist.tsx

```diff
@@ -35,25 +35,27 @@ export class LdTablist {
   @Prop() size?: 'sm' | 'lg'
 
   /** Display mode. */
-  @Prop() mode?: 'ghost' | 'brand-color'
+  @Prop() mode?: 'ghost' | 'brand-color' | 'floating'
```

Comment:
The design doesn't include anything floating-ghost or floating-on-brand related yet. It was mentioned in the requirements but has never been added to Figma. This may become a future feature.